code/gradle.properties

```diff
@@ -6,13 +6,13 @@ org.gradle.configureondemand=false
 moduleProjectName=analytics
 moduleName=analytics
 moduleAARName=analytics-phone-release.aar
-moduleVersion=2.0.0
+moduleVersion=2.0.1
 
 #bourbon-platform-android-module.gradle
 mavenRepoName=AdobeMobileAnalyticsSdk
 mavenRepoDescription=Android Analytics Extension for Adobe Mobile Marketing
 mavenUploadDryRunFlag=false
-mavenCoreVersion=2.0.0
+mavenCoreVersion=2.0.1
```

Comment:
Are there changes in Analytics 2.0.1 which require Core 2.0.1? Do we need to bump the Core version dependency?